sources/packages/web/src/views/aest/institution/ViewPendingOfferingChangeRequests.vue

```diff
@@ -0,0 +1,182 @@
+                </template>
+                <template #[`item.status`]="{ item }">
+                  <status-chip-application-offering-change
+                    :status="item.raw.status"
+                  />
+                </template>
+                <template #[`item.id`]="{ item }">
+                  <v-btn color="primary" @click="viewAssessment(item)"
+                    >View</v-btn
+                  >
+                </template>
+              </v-data-table-server>
+            </toggle-content>
+          </content-group>
+        </body-header-container>
+      </tab-container>
+    </template>
+  </full-page-container>
+</template>
+<script lang="ts">
+import { ref, defineComponent, onMounted } from "vue";
+import {
+  DEFAULT_PAGE_LIMIT,
+  DataTableOptions,
+  PaginatedResults,
+  AllInProgressOfferingChangeSummaryHeaders,
+  //   DataTableSortByOrder,
+  DEFAULT_DATATABLE_PAGE_NUMBER,
+  DataTableSortByOrder,
+} from "@/types";
+import { useFormatters } from "@/composables";
+import StatusChipApplicationOfferingChange from "@/components/generic/StatusChipApplicationOfferingChange.vue";
+import { ApplicationOfferingChangeSummaryAPIOutDTO } from "@/services/http/dto";
+import { ApplicationOfferingChangeRequestService } from "@/services/ApplicationOfferingChangeRequestService";
+import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
+import { useRouter } from "vue-router";
+
+export default defineComponent({
+  components: { StatusChipApplicationOfferingChange },
+  setup() {
+    const router = useRouter();
+    const loading = ref(false);
+    const searchCriteria = ref("");
+    const { dateOnlyLongString } = useFormatters();
+    const applications = ref(
+      {} as
+        | PaginatedResults<ApplicationOfferingChangeSummaryAPIOutDTO>
+        | undefined,
+    );
+    let currentPage = DEFAULT_DATATABLE_PAGE_NUMBER;
+    let currentPageLimit = DEFAULT_PAGE_LIMIT;
+
+    /**
+     * Load eligible applications offering change records.
+     * @param page page number, if nothing passed then {@link DEFAULT_DATATABLE_PAGE_NUMBER}.
+     * @param pageCount page limit, if nothing passed then {@link DEFAULT_PAGE_LIMIT}.
+     * @param sortField sort field, if nothing passed then api sorts with status.
+     * @param sortOrder sort oder, if nothing passed then {@link DataTableSortByOrder.DESC}.
+     */
+    const getSummaryList = async (
+      page = DEFAULT_DATATABLE_PAGE_NUMBER,
+      pageCount = DEFAULT_PAGE_LIMIT,
+      sortField = "status",
+      sortOrder = DataTableSortByOrder.DESC,
+    ) => {
+      loading.value = true;
+      applications.value =
+        await ApplicationOfferingChangeRequestService.shared.getAllInProgressApplications(
+          {
+            page,
+            sortField,
+            sortOrder,
+            pageLimit: pageCount,
+            searchCriteria: searchCriteria.value,
+          },
+        );
+      loading.value = false;
+    };
+
+    onMounted(async () => {
```

Comment:
we can also do.
` onMounted(getSummaryList())`

Reply:
That doesn't work:
![image](https://github.com/bcgov/SIMS/assets/78114138/95e178f2-a288-4e2f-bae8-b00e1227f9f1)

Reply:
@andrepestana-aot , its without the bracket, 
![image](https://github.com/bcgov/SIMS/assets/77353155/8e694338-09c7-411a-9f7d-afab9f0ca7a8)

Reply:
That works! Thanks!